src/Apps/Artist/Routes/AuctionResults/ArtistAuctionResults.tsx

```diff
@@ -214,141 +215,143 @@ const AuctionResultsContainer: React.FC<AuctionResultsProps> = ({
       0
     )
   }
+  if (results.length == 0) {
+    return <EmptyArtistAuctionResults></EmptyArtistAuctionResults>
+  } else {
+    return (
```

Comment:
From this line down there are no changes except for the indentation (because of wrapping the view in the else block).

Reply:
You don't need the `else`, in this case.

Reply:
Can I leave it in for readability reasons or is this looking weird for the JS people?